providers.tf

```
@@ -0,0 +1,26 @@
+provider "aws" {
```

```suggestion
provider "aws" {
}
```

all the role management should not be in the public module